src/go/rpk/pkg/cli/cmd/cluster/storage/recovery/start.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package recovery
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newStartCommand(fs afero.Fs) *cobra.Command {
+	var (
+		topicNamePattern string
+		wait             bool
+		pollingInterval  time.Duration
+	)
+
+	cmd := &cobra.Command{
+		Use:   "start",
```

Comment:
[question] Do we have a way to stop? or once started there is no way back?

Reply:
No way back when started